Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -5188,9 +5188,9 @@
   client:
     type: browser
     name: CCleaner
-    version: "99.0.4844.51"
+    version: 99.0.4844.51
```

Comment:
This might actually cause problems for some of the forks to other languages. If I remember correctly some yaml parsers try to handle that as number when the `"` are left away.